examples/tutorials/doc_examples/examples/all_wildcard.py

```diff
@@ -32,11 +32,11 @@ class Person(HasTraits):
 bill.age = 49
 
 # This should generate an error (must be an Int):
-print 'Attempting to assign a string to an Int trait object...\n'
+print('Attempting to assign a string to an Int trait object...\n')
 try:
     bill.age = 'middle age'
-except TraitError, c:
-    print 'TraitError: ', c, '\n'
+except TraitError as c:
+    print('TraitError: ', c, '\n')
```

Comment:
This one should probably be changed to use formatting, otherwise it's doing something different on Python 2 (printing a `tuple` rather than individual items separated by spaces). Alternatively, we could add a `from __future__ import print_function` to the top of the module.